src/MapViewController.swift

```diff
@@ -125,165 +255,391 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
+   
+   - parameter identifier: The marker to set the image for.
+   - parameter image: A UIImage to represent the marker.
+   - returns: A boolean indiciation whether or not the update succeeded.
+   */
   open func markerSetImage(_ identifier: TGMapMarkerId, image: UIImage) -> Bool {
     return tgViewController.markerSetImage(identifier, image: image)
   }
-  
+
+  /**
+   Removes an individual marker from the map.
+   
+   - parameter marker: The marker to remove.
+   - returns: A boolean indicating if the removal succeeded.
+  */
   open func markerRemove(_ marker: TGMapMarkerId) -> Bool {
     return tgViewController.markerRemove(marker)
   }
-  
+
+  /** 
+   Loads a scene file synchronously on the main thread. Use the async methods instead of these in production apps.
+   
+   - parameter path: The path to the scene file to load.
+   - throws: A MZError `apiKeyNotSet` error if an API Key has not been sent on the MapzenManager class.
+ */
   open func loadSceneFile(_ path: String) throws {
     try loadSceneFile(path, sceneUpdates: [TGSceneUpdate]())
   }
-  
+
+  /**
+   Loads a scene file synchronously on the main thread. Use the async methods instead of these in production apps.
+   
+   - parameter path: The path to the scene file to load.
+   - parameter sceneUpdates: The scene updates to make while loading the scene file.
+   - throws: A MZError `apiKeyNotSet` error if an API Key has not been sent on the MapzenManager class.
+  */
   open func loadSceneFile(_ path: String, sceneUpdates: [TGSceneUpdate]) throws {
     try tgViewController.loadSceneFile(path, sceneUpdates: updatesWithApiKeyUpdate(sceneUpdates))
   }
 
+  /**
+   Loads the scene file asynchronously. Recommended for production apps.
+   
+   - parameter path: The path to the scene file to load.
+   - parameter onSceneLoaded: Closure called on scene loaded.
+   - throws: A MZError `apiKeyNotSet` error if an API Key has not been sent on the MapzenManager class.
+  */
   open func loadSceneFileAsync(_ path: String, onSceneLoaded: OnSceneLoaded?) throws {
     try loadSceneFileAsync(path, sceneUpdates: [TGSceneUpdate](), onSceneLoaded: onSceneLoaded)
   }
-  
+
+  /**
+   Loads the scene file asynchronously. Recommended for production apps.
+   
+   - parameter path: The path to the scene file to load.
+   - parameter sceneUpdates: The scene updates to make while loading the scene file.
+   - parameter onSceneLoaded: Closure called on scene loaded.
+   - throws: A MZError `apiKeyNotSet` error if an API Key has not been sent on the MapzenManager class.
+   */
   open func loadSceneFileAsync(_ path: String, sceneUpdates: [TGSceneUpdate], onSceneLoaded: OnSceneLoaded?) throws {
     onSceneLoadedClosure = onSceneLoaded
     try tgViewController.loadSceneFileAsync(path, sceneUpdates: updatesWithApiKeyUpdate(sceneUpdates))
   }
-  
+
+  /**
+   Queue a scene update using a yaml string path. It's recommended to use the other version of this that uses the TGSceneUpdate objects.
```

Comment:
We should mention that to have the queued updates take effect they need to call `applySceneUpdates`